frontend/src/modules/dashboard/store/actions.js

```diff
@@ -157,41 +157,19 @@ export default {
   async getActiveMembers({ commit, state }) {
     const { platform, period } = state.filters;
     state.members.loadingActive = true;
-    return MemberService.list(
-      {
-        and: [
-          {
-            lastActive: {
-              gte: moment()
-                .startOf('day')
-                .subtract(
-                  period.granularity === 'day'
-                    ? period.value - 1
-                    : period.value,
-                  period.granularity,
-                )
-                .toISOString(),
-            },
-            isTeamMember: {
-              not: true,
-            },
-          },
-          ...(platform !== 'all'
-            ? [
-              {
-                activeOn: {
-                  contains: [platform],
-                },
-              },
-            ]
-            : []),
-        ],
-      },
-      'activityCount_DESC',
-      5,
-      0,
-      false,
-    )
+    return MemberService.listActive({
+      platform: platform !== 'all' ? platform : [],
```

Comment:
Just noticed another issue while testing this locally.
When you change the period all good 👌 but if you change the platform filter to a specific platform (you need to have more than 1 integration for it to appear), the list of members is not updated.
The issue is with the way you are passing `platform` to the `listActive` method.

If you notice there `platform` must be an array of objects. So for this to work you would need to update this to something like this:
```
platform: platform !== 'all' ? [{ value: platform }] : [],
```
![Screenshot 2023-04-21 at 17 34 18](https://user-images.githubusercontent.com/20134207/233688749-46f3bb3c-31fd-42f7-9af7-76992f3f386f.png)

Reply:
Pushed the changes, thanks for letting me know.

I'm marking the conversation as resolved and also will ensure no bugs are introduced next time with the code 🙂